docs/source/api_reference/skspatial.objects.Circle.rst

```diff
@@ -13,4 +13,5 @@ Methods
    ~skspatial.objects.Circle.area
    ~skspatial.objects.Circle.circumference
    ~skspatial.objects.Circle.intersect_line
+   ~skspatial.objects.Circle.best_fit
```

Comment:
Could we move this line up two so the lines are in alphabetical order?

Reply:
For sure, done! Now the they are in alphabetical order.

I apologize, I can't find the file `CONTRIBUTORS.md`. Do I need to create one? Thanks!

Reply:
My mistake, the file is `AUTHORS.rst`. You can add your name in the Contributors section.

Reply:
Done!